src/cmds/init.py

```diff
@@ -36,11 +49,20 @@ def migrate_from(old_root, new_root, manifest):
             mkdir(new_path.parent)
             shutil.copy(old_path, new_path)
         else:
+            not_found.append(f)
             print(f"{old_path} not found, skipping")
+    # update config yaml with new keys
+    config = load_config(new_root, "config.yaml")
+    config_str = initial_config_file("config.yaml")
+    default_config = yaml.load(config_str)
+    dict_add_new_default(config, default_config)
```

Comment:
This rewrites the config file, which means comments and ordering is lost, right?

Reply:
Yes